ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml

```diff
@@ -551,6 +551,11 @@ tests:
           cpu: 100m
           memory: 600Mi
     workflow: ipi-azure
+- as: e2e-telco5g
+  cluster: build01
```

Comment:
There are a few jobs that pin their build cluster like this (e.g. #21425), but I don't see anything in this commit message or PR that explains why these jobs need to be pinned to `build01`.  If there is a reason, can you explain it in the commit message?  If there is no reason, can you drop this pin?

Reply:
I think this is related to bastion running (only?) in this cluster
@johnbieren can you verify?
even if thats the case, probably best to enable it on all clusters if possible..

Reply:
done, added to commit message.